client/src/Try/API.purs

```diff
@@ -135,13 +98,13 @@ get url = ExceptT $ AX.get AXRF.string url >>= case _ of
     pure $ Right body
 
 -- | POST the specified code to the Try PureScript API, and wait for a response.
-compile :: String -> String -> ExceptT String Aff (Either (NonEmptyList ForeignError) CompileResult)
-compile endpoint code = ExceptT $ AX.post AXRF.json (endpoint <> "/compile") (Just requestBody) >>= case _ of
+compile :: String -> String -> ExceptT String Aff (Either String CompileResult)
+compile endpoint code = ExceptT $ AX.post AXRF.json (endpoint <> "/compile") requestBody >>= case _ of
   Left e ->
     pure $ Left $ printError e
   Right { status } | status >= StatusCode 400 ->
     pure $ Left $ "Received error status code: " <> show status
   Right { body } ->
-    pure $ Right $ runExcept (decode (unsafeToForeign body))
+    pure $ Right $ decodeJson body
   where
-  requestBody = AXRB.String code
+  requestBody = Just $ AXRB.Json $ encodeJson code
```

Comment:
Wondering if the CORS issue reported in #215 is (unexpectedly) related to using `AXRB.Json` instead of `AXRB.String`.

Reply:
Wow, great find -- it totally is. Changing this to `AXRB.String $ unsafeCoerec (encodeJson code)` works as expected. I don't know why -- going to look a little deeper.

Reply:
Well, on reflection the code isn't actually JSON. It's just a string. Encoding it runs through a `stringify` call, which isn't what we want; I still don't know why this would cause a CORS issue (perhaps it's a red herring?) but regardless this wasn't a correct change to make in the first place.